### PR TITLE
Feat/add username in response#127

### LINKED ISSUE #127: Add user name in response
Replace in response user_id by user: {id, name} (where name is user login) in end-point http://dev2.badgerdoc.com/api/v1/annotation/docs#/Manual%20Annotation%20Tasks/get_tasks_tasks_get

### PR DESCRIPTION
**Pull request by thinklab**
closes #127

### CODE REVIEW

File: annotation/app/microservice_communication/search.py [hunk 1]
```diff
@@ -193,6 +194,10 @@ def expand_response(
             "id": response_task.pop("job_id"),
             "name": job_names.get(task.job_id, None),
         }
+        response_task["user"] = {
+            "id": response_task.get("user_id"),
+            "name": user_logins.get(response_task.get("user_id")),
+        }
```
Comment (spiridonovfed) [inline annotation/app/microservice_communication/search.py:200]: Can we get an 'id' by response_task.user_id?
And 'name': userl_logins.response_task.user_id?

File: annotation/app/microservice_communication/user.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            f"{callback_url}/{user_id}",
+            headers={
+                "X-Current-Tenant": tenant,
+                "Authorization": f"Bearer {token}",
+            },
+            timeout=5,
+        )
+        if user_response.status_code != 200:
+            raise GetUserInfoException(user_response.text)
+        return user_response.json().get("username")
```
Comment (spiridonovfed) [inline annotation/app/microservice_communication/user.py:30]: I believe it is important to let API clients know, that request to 'users' microservice has failed.
I would probably use: 
raise GetUserInfoException(f"Failed request to 'users' microservice: {user_response.text}")

File: annotation/app/microservice_communication/user.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+def get_user_logins(
+    tasks: List[ManualAnnotationTask], tenant: str, token: str
+) -> Dict[str, str]:
+    """
+    Return list of logins for provided tasks from users microservice.
+    """
+    user_ids = [_.user_id for _ in tasks]
+    return {
```
Comment (spiridonovfed) [inline annotation/app/microservice_communication/user.py:42]: In user_ids = [_.user_id for _ in tasks] you obviously use tasks instances.

It is more readable this way:
[task.user_id for task in tasks]

File: annotation/app/microservice_communication/search.py (hunk 1 above)
Comment (thinklab) [reply annotation/app/microservice_communication/search.py:200]: No we can't - .user_id not always there.

**Pull request merged**